cupy/sparse/csr.py

```diff
@@ -60,6 +62,10 @@ def get(self, stream=None):
         return scipy.sparse.csr_matrix(
             (data, indices, indptr), shape=self._shape)
 
+    def _convert_dense(self, x):
+        m = cusparse.dense2csr(x)
```

Comment:
Yes, if we take the case. But I often separate layers too strict, the current implementation is acceptable enough for me. We can get forward with that.